configure.ac

```diff
@@ -174,6 +174,95 @@ AC_ARG_ENABLE(oclblender,
 AM_CONDITIONAL(BUILD_OCL_BLENDER,
     [test "x$enable_oclblender" = "xyes"])
 
+APPS=""
+
+dnl decoder
```

Comment:
why we need this, could we just compile all things?

Reply:
because libyami may not compile decoder or encoder, so we have to disable decoder in case that libyami disable all decoders. In fast-boot case, users only need decoder, so we should disable encoder and other functional code except for decoder.